### PR TITLE
Add nixidy

### PR DESCRIPTION
**Pull request by arnarg**
Manage an entire Kubernetes cluster like it's NixOS, then have CI generate plain YAML manifests for ArgoCD. It's useful for myself and might be for others as well :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,6 +164,7 @@
 ## DevOps
 
 * [Makes](https://github.com/fluidattacks/makes) - A Nix-based CI/CD pipeline framework for building, testing, and releasing projects in any language, from anywhere.
+* [nixidy](https://github.com/arnarg/nixidy) - Kubernetes GitOps with nix and Argo CD.
```
Comment (cyntheticfox) [inline README.md:167]:
```suggestion
* [nixidy](https://github.com/arnarg/nixidy) - Kubernetes GitOps with Nix and Argo CD.
```

Just fix the capitalization of Nix and it's good

**Review by cyntheticfox — CHANGES_REQUESTED** [review]
Small casing change to meet `awesome` project requirements, and it'll be good

**Review by cyntheticfox — APPROVED** [review]
Looks good, and link fail is unrelated

**Comment by arnarg** [comment]
Perfect thanks!

**Pull request merged**